easybuild/easyblocks/i/icc.py

```diff
@@ -155,22 +155,3 @@ def make_module_req_guess(self):
                     dirmap[k].append(v2)
 
         return dirmap
-
-    def make_module_extra(self):
```

Comment:
besides the issue of breaking `icpc` with having this set, this is not the right location for it either

when `make_module_extra` is called, the modules for the dependencies are not loaded anymore, so running `g++`  means using the *system* GCC, resulting in this even when `icc` depends on a EB-provided `GCC`:

```
$ module show icc 2>&1 | grep GXX
setenv           GXX_ROOT /usr/lib/gcc/x86_64-redhat-linux/4.4.7/ 
```